src/main/java/org/llorllale/cactoos/matchers/HasLines.java

```diff
@@ -100,22 +99,29 @@ public HasLines(
         final Collection<String> lns
     ) {
         super();
-        this.fnc = new UncheckedBiFunc<>(fnc);
-        this.separator = new UncheckedScalar<>(stor);
```

Comment:
@dgroup can you do me a favor and rename `stor` to something like `sep`?

Reply:
@llorllale sure :)